salt/output/json_out.py

```diff
@@ -54,7 +54,7 @@ def output(data):
     '''
     try:
         if 'output_indent' not in __opts__:
-            return json.dumps(data, default=repr, indent=4)
+            return json.dumps(data, default=repr, indent=4, encoding='latin1')
```

Comment:
Ok, will post the rest later, but here is allready the locale_infopart.
        locale_info:
            ----------
            defaultencoding:
                cp1252
            defaultlanguage:
                en_US